x-pack/plugins/security/server/user_profile/user_profile_service.ts

```diff
@@ -315,12 +315,13 @@ export class UserProfileService {
       return null;
     }
 
+    let body;
     try {
-      const body = await clusterClient.asInternalUser.security.getUserProfile({
+      // @ts-expect-error Invalid response format.
```

Comment:
__note:__ ES client isn't updated yet to support the latest response/request signature. I'll try to update it in the follow-up next week.